pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -193,18 +191,28 @@ async def run(self):
                         justifications.append(justification)
 
             # Verify attached bugs
-            advisories = list(filter(lambda ad: ad > 0, impetus_advisories.values()))
-            if advisories:
-                if self.skip_attached_bug_check:
-                    logger.info("Skip checking attached bugs.")
+            if self.skip_attached_bug_check:
+                logger.info("Skip checking attached bugs.")
+            else:
+                major, minor = util.isolate_major_minor_in_group(self.group)
+                next_minor = f"{major}.{minor + 1}"
+                logger.info("Checking if %s is GA'd...", next_minor)
+                cincinnati = CincinnatiAPI()
+                graph_data = await cincinnati.get_graph(channel=f"fast-{next_minor}")
+                no_verify_blocking_bugs = False
+                if not graph_data.get("nodes"):
+                    logger.info("%s is not GA'd. Blocker Bug check will be skipped.", next_minor)
```

Comment:
Once `elliott verify-attached-bugs` is not broken I'd like to have a quick look and see what happens if we don't skip this. I'd prefer we don't unless we have to, but I'm not sure we remember the reason why we did this.